<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>object_list_frame</class>
 <widget class="QFrame" name="object_list_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="Z"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="position_hint">
      <property name="text">
       <string>X Y Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="X">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="Y"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="customSpinBox1"/>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="customSpinBox2"/>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="customSpinBox3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
